app/swagger/schemas/forms.rb

```diff
@@ -38,13 +38,6 @@ class Forms
                 key :example, 'Information for Pre-Complaint Processing'
               end
 
-              property :first_issued_on do
-                key :description, 'The date the form first became available'
-                key :type, :string
-                key :example, '2012-01-01'
-                key :format, 'date'
-              end
-
```

Comment:
Should I also update the cassette to reflect this property being removed? It's always `null` from Lighthouse right now.

Reply:
If its part of the response, it should probably stay in the cassette

Reply:
Sounds good!